.github/workflows/test-unit-shared.yml

```diff
@@ -0,0 +1,33 @@
+name: Integration tests - License Kit (Node)
```

Comment:
The name should be updated to sth like: "Unit tests - React Native Legal Shared package"

Reply:
Good find, thanks. Addressed in 8aeae76